bam_ampliconclip.c

```diff
@@ -102,12 +104,23 @@ int load_bed_file_pairs(char *infile, int get_strand, int sort_by_pos,
         if (get_strand) {
             char strand;
 
-            if (sscanf(line.s, "%*s %"SCNd64" %"SCNd64" %*s %*s %c", &left, &right, &strand) != 3) {
+            if (sscanf(line.s, "%255s %"SCNd64" %"SCNd64" %*s %*s %c",
```

Comment:
So, there are two sscanf statements depending on whether the strand is being read or not.  Should both be changed to handle the ref name.

Reply:
Also, in my copyright police role, the years need changing.

Reply:
Probably yes - I didn't notice the second sscanf.  I'll do both.